qingstor/sdk/error.py

```diff
@@ -43,3 +43,21 @@ def __str__(self):
             " is not allowed, should be one of ",
             ", ".join(self.allowed_values)
         ])
+
+
+class PartTooSmallError:
+    def __init__():
+        pass
+    def __str__(self):
+        return repr("Invalid Request!\n \
```

Comment:
Ugly string here.
Do not use `\` for line breaking, do like previous one:

```python
    def __str__(self):
        return "".join([
            self.parameter_name, " value ", self.parameter_value,
            " is not allowed, should be one of ",
            ", ".join(self.allowed_values)
        ])
```

And it's too long for an error.